src/app/workflow/register-workflow-modal/register-workflow-modal.component.html

```diff
@@ -49,7 +49,34 @@ <h4 mat-dialog-title>Register Workflow</h4>
     </mat-step>
     <mat-step>
       <ng-template matStepLabel>Create a workflow</ng-template>
-      <div *ngIf="selectedOption?.value == 0">
+      <div *ngIf="selectedOption?.value === 0">
+        <p>
+          Navigate to GitHub to install our GitHub app on your repositories/organizations. See our
+          <a href="{{ Dockstore.DOCUMENTATION_URL }}/getting-started/github-apps.html" target="_blank" rel="noopener noreferrer">
+            documentation <mat-icon>open_in_new</mat-icon>
+          </a>
+          for how to use GitHub Apps to automatically sync your workflows with GitHub.
+        </p>
+        <div fxLayout="column" fxLayoutAlign="center center" fxLayoutGap="2rem" class="mb-2">
+          <a
+            mat-flat-button
+            type="button"
+            color="primary"
+            [href]="gitHubAppInstallationLink$ | async"
+            target="_blank"
+            rel="noopener noreferrer"
+            matTooltip="Manage on GitHub"
+            ><mat-icon>add</mat-icon> Manage Installations on GitHub</a
```

Comment:
"Installations" sounds odd to me. Is it because you install the app into your GitHub orgs/repos? No alternative suggestions at this point though.

Reply:
Ya, that was my logic. I agree there is probably a better way to phrase it but unsure what that would be.

Reply:
`Installations` is used pretty consistently in the GItHub documentation. Perhaps what is needed here is to clarify that it is "our" app being installed to their repos/orgs?

Reply:
Manage Connected Repositories on GitHub?